pkg/user/repository.go

```diff
@@ -115,3 +126,17 @@ func (r repository) update(user *model.User) (*model.User, error) {
 
 	return user, nil
 }
+
+func (r repository) resetPassword(user *model.User) (*model.User, error) {
+	updatedUser := model.User{
+		Password:      user.Password,
+		PasswordToken: sql.NullString{String: "", Valid: false},
+	}
+
+	err := r.db.Model(&user).Select("Password", "PasswordToken").Updates(updatedUser).Error
```

Comment:
Most other places in the file we add new lines between the method calls for the sake of readability. I think that should be done here as well